src/unstract/sdk/tool/executor.py

```diff
@@ -62,9 +60,13 @@ def execute_run(self, args: argparse.Namespace) -> None:
             f"SDK Version: {get_sdk_version()}, "
             f"adapter Version: {get_adapter_version()}"
         )
-        self.tool.run(
-            settings=settings,
-            input_file=self.tool.get_input_file(),
-            output_dir=self.tool.get_output_dir(),
-        )
+        try:
+            self.tool.run(
+                settings=settings,
+                input_file=self.tool.get_input_file(),
+                output_dir=self.tool.get_output_dir(),
+            )
+        except Exception as e:
+            self.tool.stream_error_and_exit(f"Error while running tool: {str(e)}")
```

Comment:
Would be better if we add some logger.error with traceback.

Reply:
Addressed

PF screenshot of tool container
![image](https://github.com/user-attachments/assets/df645f19-2fd4-4474-b587-83cd5c868d63)